src/Microsoft.Health.Dicom.Client/Models/OperationReference.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Dicom.Client.Models
+{
+    /// <summary>
+    /// Represents a reference to an existing long-running oepration.
+    /// </summary>
+    public class OperationReference
```

Comment:
At some point we should definitely have a separate shared lib that contains our API contracts. That is way too big of an ask to do in this PR but something to remember.